### PR TITLE
add value prop images

### PR DESCRIPTION
**Pull request by kcvan**
## Description
Why did you write this code?
add value prop images and fade component to containers
## How to Test
- [ ] Visit /work
- [ ] verify things fadein and out and value prop images are there

Which devices did you test on?
- [x] Chrome on Mac
- [ ] Chrome on PC
- [ ] Firefox on Mac
- [ ] Firefox on PC
- [ ] Safari iPhone
- [ ] Chrome Android

## REVIEWERS:
Check against these principles:

### High level
Does this code need to be written?
What are the alternatives?
Will this implementation become a support issue?
How much error margin does this solution have?

### Code
* Does the code follow industry standards?
JS: https://github.com/airbnb/javascript
React: https://github.com/airbnb/javascript/tree/master/react
https://github.com/vasanthk/react-bits
Documentation headers: http://usejsdoc.org/index.html

* Is there duplicated code? Can it be refactored into a shared method?
* Is the code consistent with our project?
* Are there unit tests? Do they test the states?
* Is the person refactoring another developer's code? If possible, did the original developer approve?

### Variables/Naming:
* Would the variable type led to future edge cases?
* Are the variable naming clear? Would the value contain something other than what the name describes.

### Security
* Can this be hacked or abused by the user?

## Further Work
Will this have future work?

## Learnings
Did you learn anything here you want to share with the team?

### CODE REVIEW

File: src/pages/home/Home.tsx [hunk 1]
```diff
@@ -81,7 +82,7 @@ const Home = () => {
 };
 
 const HomeGuests = () => (
-  <>
+  <Fade>
```
Comment (jerminatorhits) [inline src/pages/home/Home.tsx]: injec `Fade` via `<Container tag={...} />`

File: src/pages/home/Home.tsx [hunk 2]
```diff
@@ -116,11 +117,11 @@ const HomeGuests = () => (
         </Row>
       </Container>
     </Container>
-  </>
+  </Fade>
 );
 
 const HomeHosts = () => (
-  <>
+  <Fade>
```
Comment (jerminatorhits) [inline src/pages/home/Home.tsx]: here too

File: src/pages/home/Home.tsx (hunk 1 above)
Comment (kcvan) [reply src/pages/home/Home.tsx]: it wont work for here since there are two elements being returned

File: src/pages/home/Home.tsx (hunk 2 above)
Comment (kcvan) [reply src/pages/home/Home.tsx]: 👍

File: src/pages/home/Home.tsx (hunk 1 above)
Comment (jerminatorhits) [reply src/pages/home/Home.tsx]: 👍

**Pull request merged**